deckhouse-controller/pkg/apis/deckhouse.io/validation/validate_module.go

```diff
@@ -42,9 +42,9 @@ func moduleValidationHandler() http.Handler {
 	wh, _ := kwhvalidating.NewWebhook(kwhvalidating.WebhookConfig{
 		ID:        "module-operations",
 		Validator: vf,
-		Logger:    validationLogger,
+		Logger:    nil,
```

Comment:
Let's leave a comment about why the logger is nil to hint that nil is set here intentionally and this should not be changed.